content/departments/sales/tools/referral_and_reseller_policy.md

```diff
@@ -0,0 +1,94 @@
+## Referral Program Summary
```

Comment:
This is a heading level 2, but the first heading always _has_ to be a heading level 1 ("Heading 1" in Google docs).

All the other headings then need to be demoted one level down too (3->2, etc)